src/python/pants/notes/2.0.x.rst

```diff
@@ -3,7 +3,62 @@
 
 This document describes releases leading up to the ``2.0.x`` ``stable`` series.
 
-See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release.
+See https://www.pantsbuild.org/v2.0/docs/release-notes-2-0 for an overview of the changes in this release, and
+https://www.pantsbuild.org/docs/plugin-upgrade-guide for a plugin upgrade guide.
+
+2.0.0rc2 (10/21/2020)
+---------------------
+
+User API Changes
+~~~~~~~~~~~~~~~~
+
+* Rename pex-binary/python-distribution to avoid naming collisions. (Cherry-pick of #11007) (#11010)
+  `PR #11010 <https://github.com/pantsbuild/pants/pull/11010>`_
+
+* Increase default `[python-setup].resolver_jobs` to `cpu_count / 2` (Cherry-pick of #11006) (#11011)
+  `PR #11011 <https://github.com/pantsbuild/pants/pull/11011>`_
+
+* Include `<PYENV>` in `[python-setup].interpreter_search_paths` default (Cherry-pick of #10998) (#11008)
+  `PR #11008 <https://github.com/pantsbuild/pants/pull/11008>`_
+
+* Upgrade Pex to 2.1.20 (Cherry-pick of #11014) (#11016)
+  `PR #11016 <https://github.com/pantsbuild/pants/pull/11016>`_
+
+Plugin API Changes
+~~~~~~~~~~~~~~~~~~
+
+* Validate that option scopes and target aliases don't collide. (Cherry-pick of #11004) #11015
+  `PR #11015 <https://github.com/pantsbuild/pants/pull/11015>`_
+
+New Features
+~~~~~~~~~~~~
+
+* Allow changing the versioning scheme for `python_distribution` first-party dependencies (Cherry-pick of #10977) (#10989)
+  `PR #10989 <https://github.com/pantsbuild/pants/pull/10989>`_
+
+* Add `[python-setup].resolver_http_cache_ttl` option (Cherry-pick of #10997) (#11002)
+  `PR #11002 <https://github.com/pantsbuild/pants/pull/11002>`_
+
+* Allow changing the URL to download tools like PEX and Protoc (Cherry-pick of #11013) (#10...)
+  `PR #10... <https://github.com/pantsbuild/pants/pull/10...>`_
```

Comment:
In anticipation of https://github.com/pantsbuild/pants/pull/11013 landing, then being cherry-picked. If we can't get it in time for rc2, we can put in rc3 on Friday.

cc @thamenato